collections/_article/funding-breakdown-and-hiring-process.md

```diff
@@ -0,0 +1,102 @@
+
+The new donation platform is mostly ready; you can already use it to set up a recurring donation via PayPal if you would like! In the coming weeks, we expect to finalize the donation platform and we will have a big announcement post once it is 100% finished.
+
+We will keep the Patreon operational for the foreseeable future as we heavily rely on the current donations we receive through it. But for future donations, please consider using the Development Fund as it results in more of your donation going towards development.
+
+### Diversification of funding sources
+
+With the new Godot Foundation, it is expected that the Godot Project should be able to further diversify its funding, going into areas such as:
+
+- Paid asset store (taking a percentage of sales).
+- Merchandise.
+
+These are still in the early planning phases as our priority until now has been to get the donation platform operational. To be clear, we do not anticipate that we will be able to sustain the project through the asset store and merchandise alone as we will intentionally operate them with very thin margins. We are a not-for-profit project first and foremost and we will not compromise that.
+
+### Transparency Reports
+
+During 2023, the transition from Software Freedom Conservancy to the Godot Foundation will take place. Existing contracts with the SFC will end and new contracts with the Godot Foundation will be signed to ensure continuity.
+
+The SFC publishes its own [transparency reports](https://sfconservancy.org/about/transparency/) at the end of every year with detailed information on funding sources and use of funds, but as they manage several projects, they don't go into the specifics of each one for every spending category. Godot will be included in the SFC’s annual report for 2022 and 2023, but the Godot Foundation will be responsible for publishing its own annual reports for 2023 onward.
+
+The Godot Foundation will publish its own transparency reports in 2024 covering its inception in 2022 until the end of 2023 based on its own funding and cash flows. This will overlap with the SFC's 2022 and 2023 reports. As the report will cover Godot exclusively, it can cover more details than what the SFC reports can cover. Particularly, interested readers can expect to find clear breakdowns of sources of funding and categories of expenses.
+
+While it may be desirable to publish this information more often, please understand that these reports have to be audited to comply with local law and can be expensive and time consuming to prepare. We will. however, be able to post up to date information about our monthly funding thanks to the new donation platform.
```

Comment:
```suggestion
While it may be desirable to publish this information more often, please understand that these reports have to be audited to comply with local law and can be expensive and time consuming to prepare. We will, however, be able to post up to date information about our monthly funding thanks to the new donation platform.
```